resources/frontend/wp-convertkit.js

```diff
@@ -1,34 +1,38 @@
 jQuery(document).ready(function($) {
 
     // Manage visit cookie
-    var subscriber_id = $.cookie( 'ck_subscriber_id' );
+    var subscriber_id = parseInt($.cookie('ck_subscriber_id')) ? parseInt($.cookie('ck_subscriber_id')) : false;
```

Comment:
Not a fan of ternary operator, but if you like it, I'm OK with it. it would be nice to have some guideline as to when to use it.

Reply:
@pothibo I am pretty partial to it, but that's fairly subjective/personal.

What would make sense to you as a guideline? Anything you can point me to (just guidelines in general, not ternary-specific) for the ConvertKit app I can look at?

Reply:
Nothing really, I think it was meant more as a general discussion/direction as to the guidelines to writing code in the Wordpress plugin.

We have people on either side of the fence on the JS/Ruby side so it's still up for debate. We are , however, starting to use more Rubocop (linter) to set the guideline there. I think my comment was more of a food for thoughts related to how we want this project to look in the future.

Like when we get to the documentation/linter aspect of it, we should think about stuff like ternary operator and make a decision on it. (No embedded ternary, no ternary at all, etc.)